### ISSUE TITLE
General issues with pressure tubes

### ISSUE THREAD

**Issue by MuteTiefling** [TeamPneumatic/pnc-repressurized#460]
*For feature requests, just erase this template and clearly describe the feature you'd like to see*

### Minecraft Version
1.15.2
### Forge Version
forge-31.1.74
### Mod Version
pneumaticcraft-repressurized-1.15.2-1.1.1-6
### Describe your problem, including steps to reproduce it

Pressure tubes aren't connecting in ways that I've come to expect. Going around corners sometimes leaves them venting air and adding some inline modules is causing them to disconnect depending on orientation. I'm also encountering issues where I wrench a section closed but it continues to vent air. 

### Any other comments?

This displays a few of the issues. Only mods here are JEI, journeymap, Silent's Mechanisms (as this is the same test world as previous), and PNC.

Video https://streamable.com/ekil7w

**Comment by desht** [comment]
I've seen occasional occurrences of this in my dev world, but they've been rare and thus very hard to track down.  If you've found a reliable way of reproducing it, that would be useful :)

**Comment by MuteTiefling** [comment]
The one in the video has been pretty consistent. It did the same in my e5 world and my test pack.

Basically I just put the module on at a right angle so that the pipe going into it goes in from the side rather than the back. That immediately disconnects the inbound pipe and then causes that section to bug out as seen, where wrenching it doesn't behave correctly anymore.

I had a few other instances I couldn't recreate, but it's been pretty consistently involving right angle connections.

**Comment by desht** [comment]
OK, for that case (placing a regulator at right angles), I did - deliberately - change it so that inline modules (which is the regulator and flow meter) can no longer be connected to from the side.  It never worked properly in 1.12 anyway (it connected, but regulation never worked correctly for tubes coming out of the side of a regulator), and it doesn't make much sense either.  So that's deliberate.

Having said that, the current behaviour is rather unintuitive, in that you're able to place the second regulator in the way that you did.  I will change that so the regulator can only go on the open section of tube, i.e. the way the tube is pointing.

I think I've also spotted why wrenched sections sometimes still vent, and that should be an easy enough fix...
